### PR TITLE
Changed the showcaseview

### PR DESCRIPTION
**Pull request by Ananthusubramanian**
Solution to issue #1506 

Changed the showcaseview so that it closed on touch outside. 

Fix: Now the  showcaseview  closes on touch outside
Changes:Changed the showcaseview

### CODE REVIEW

**Comment by anantprsd5** [comment]
@Ananthusubramanian Please read the contribution guidelines. Follow the best practices.

**Comment by anantprsd5** [comment]
@Ananthusubramanian Please follow the pull request template

File: app/src/main/java/org/fossasia/phimpme/base/BaseActivity.java [hunk 1]
```diff
@@ -86,6 +86,7 @@ public void onShow(MaterialShowcaseView itemView, int position) {
                         .setTarget(nav_cam)
                         .setDismissText(getResources().getString(R.string.ok_button))
                         .setContentText(getResources().getString(R.string.camera_button))
+                        .hideOnTouchOutside()
```
Comment (mayank8318) [inline app/src/main/java/org/fossasia/phimpme/base/BaseActivity.java]: Try using using `.setDismissOnTouch(true)` instead

**Comment by Ananthusubramanian** [comment]
I reviewed it.

File: app/src/main/java/org/fossasia/phimpme/base/BaseActivity.java (hunk 1 above)
Comment (Ananthusubramanian) [reply app/src/main/java/org/fossasia/phimpme/base/BaseActivity.java]: Did it and worked.

**Comment by Ananthusubramanian** [comment]
@anantprsd5 I changed the code please check it.

**Comment by anantprsd5** [comment]
You have deleted your repository. Not possible to merge. Thanks.

**Comment by anantprsd5** [comment]
Please make another PR and don't delete the repo from which the PR is sent.

**Comment by Ananthusubramanian** [comment]
Sorry ,I have created a new PR Changed ShowcaseView #1619

**Comment by Ananthusubramanian** [comment]
Please look into it.

**Pull request closed without merging**